<launch>
  <node name="force_sensor_node" pkg="phidget_ros_drivers" type="phiget_force_sensor.py" output="screen">
    <rosparam param="loadcell_mapping">[726AR, 811AR, 792AR, 782AR]</rosparam>
    <param name="calibration_file" value="$(find phidget_ros_drivers)/config/loadcell_calibration.yaml" />
  </node>
</launch>
